src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
```

Comment:
All files are in same package, this makes navigation harder. Please add subpackages (model, service etc.)

Reply:
Ok